src/syntax/rules.rkt

```diff
@@ -696,53 +800,67 @@
       [atan-tan-s . (<= (fabs x) (/ PI 2))]))
 
   (define *skip-tests*
-    ;; All these tests fail due to underflow to 0 and are irrelevant
-    '(exp-prod pow-unpow pow-pow pow-exp
-               asinh-2 tanh-1/2* sinh-cosh
-               hang-p0-tan hang-m0-tan))
+    (append
+      ;; All these tests fail due to underflow to 0 and are irrelevant
+      ;; Posit tests may have unnaceptable error due to lack of
+      ;; representable numbers
+      '(exp-prod pow-unpow pow-pow pow-exp
+        asinh-2 tanh-1/2* sinh-cosh
+        hang-p0-tan hang-m0-tan erf-odd erf-erfc erfc-erf
+        p16-flip-- insert-quire-sub sqrt-sqrd.p16)))
 
   (for* ([test-ruleset (*rulesets*)]
          [test-rule (first test-ruleset)]
          #:unless (set-member? *skip-tests* (rule-name test-rule)))
     (parameterize ([bf-precision 2000])
-    (test-case (~a (rule-name test-rule))
-      (match-define (rule name p1 p2 _) test-rule)
-      ;; Not using the normal prepare-points machinery for speed.
-      (define fv (free-variables p1))
-      (define valid-point?
-        (if (dict-has-key? *conditions* name)
-            (eval-prog `(λ ,fv ,(dict-ref *conditions* name)) 'bf)
-            (const true)))
-
-      (define (make-point)
-        (for/list ([v fv])
-          (match (dict-ref (rule-itypes test-rule) v)
-            ['real (sample-double)]
-            ['bool (if (< (random) .5) false true)]
-            ['complex (make-rectangular (sample-double) (sample-double))])))
-      (define point-sequence (sequence-filter valid-point? (in-producer make-point)))
-      (define points (for/list ([n (in-range num-test-points)] [pt point-sequence]) pt))
-      (define prog1 (compose ->flonum (eval-prog `(λ ,fv ,p1) 'bf)))
-      (define prog2 (compose ->flonum (eval-prog `(λ ,fv ,p2) 'bf)))
-      (with-handlers ([exn:fail:contract? (λ (e) (eprintf "~a: ~a\n" name (exn-message e)))])
-        (define ex1 (map prog1 points))
-        (define ex2 (map prog2 points))
-        (define errs
-          (for/list ([v1 ex1] [v2 ex2])
-            ;; Ignore points not in the input or output domain
-            (if (and (ordinary-value? v1) (ordinary-value? v2))
-                (ulps->bits (+ (abs (ulp-difference v1 v2)) 1))
-                #f)))
-        (when (< (length (filter identity errs)) 100)
-          (eprintf "Could not sample enough points to test ~a\n" name))
-        (define score (/ (apply + (filter identity errs)) (length (filter identity errs))))
-        (define max-error
-          (argmax car (filter car (map list errs points ex1 ex2 errs))))
-        (with-check-info (['max-error (first max-error)]
-                          ['max-point (map cons fv (second max-error))]
-                          ['max-input (third max-error)]
-                          ['max-output (fourth max-error)])
-                         (check-pred (curryr <= 1) score)))))))
+    (with-check-info (['rule test-rule])
```

Comment:
The change here from `with-check-info` to `test-case` is a merge error, you want the version with `test-case`, from `develop`.